docs/setup_eecs281.md

```diff
@@ -132,17 +196,70 @@ hello world!
 ## Parsing command line arguments and options
 Edit your main program (e.g., `main.cpp`) to parse command line options and print them.
 
+If you are using Visual Studio, follow the  [`getopt` library](#getopt-library) instructions.
+
+Copy this sample code into your main program.
+
 ```c++
```

Comment:
I made a bunch of changes to this.  LMK what you think.

Reply:
They all look good to me!